slaves/linux-cross/arm-linux-gnueabihf.config

```diff
@@ -0,0 +1,577 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Crosstool-NG Configuration
+#
+# Generated with crosstool-ng-1.22.0's `ct-ng menuconfig` command using the default configuration
+# plus the following modifications
+#
+# Target options > Target Architecture = arm
+# Target options > Architecture level = armv7-a -- (*)
```

Comment:
> I think that exporting CC_armv7_unknown_linux_gnu will be enough to hijack the entire rustbuild build system, but I haven't tested that yet.

The CC_ does work for rustbuild! I've tested it before. Actually, my question was aimed at the old/Makefile-based build system because AFAIK that's where we currently build libstd for arm. Unless the plan is to switch that build job to rustbuild when this new docker image is ready?